addons/SpellBook/SpellBook.lua

```diff
@@ -0,0 +1,444 @@
+_addon.name     = 'SpellBook'
+_addon.author   = 'SigilBaram'
+_addon.version  = '0.0.1'
```

Comment:
Is it really that pre-alpha? :D

Reply:
Yeah... Made it version 1.0.0 since it's ready for release or whatnot. xD